lib/index.js

```diff
@@ -151,6 +152,7 @@ class NewsArticle extends BaseAsset {
     to_metadata() {
         const metadata = super.to_metadata();
         metadata['document'] = this._document.toString();
+        metadata['synopsis'] = this._synopsis.toString();
```

Comment:
@sgnn7 I think I know what you're asking...which is, if there is no 'synopsis' value set, `this._synopsis.toString()` will return undefined and throw an error. I experienced something like this when I was working with the `document` field (I didn't pass it a value) and I got some unreturned promise errors.

Assuming this is what your'e getting at, would some conditional logic in the ingester to set synopsis to empty string (if no synopsis, synopsis = "") be a workaround? Furthermore, would similar types of logic needed to be placed in the code for the various other fields, like `document`? But because I don't see any such logic elsewhere, now I'm wondering if this line of thinking is accurate.